### DISCUSSION TITLE
Manual Mesh Leveling in self compiled firmware

### DISCUSSION THREAD

**Discussion by Conboss** [Jyers/Marlin#476] [Q&A] [answered]
Hello, very interested in getting a print going on this firmware! 

I've downloaded the firmware with 3X3 mesh leveling, however this does not account for my hero me, which has a -9mm offset. I have to compile my own firmware for this, but that's no issue. I'm trying to figure out which options to change to get this mesh system working from my own build. 

Thanks! 

edit: after a little more research, I've found most of the edits to do, however when I compile it tells me it needs a programmable LCD to complete.

**Comment by Jyers** [comment, marked as answer]
"Configuration Files/E3V2 Templates" has all the base files you need for the different versions. There you can find the config files for manual mesh leveling.
